README.md

```diff
@@ -16,6 +16,7 @@ MailerSend Node.js SDK
     + [Advanced personalization](#advanced-personalization)
     + [Simple personalization](#simple-personalization)
     + [Send email with attachment](#send-email-with-attachment)
+    + [Send bulk emails and check status](#send-bulk-emails-and-check-status)
```

Comment:
Based on the doc, there should be two separate methods: 

- Send bulk emails
- Get bulk request status

Also, It was instructed by @Agathe-Brusset to follow the same 'title' for the feature as it is in the doc.

https://docs.google.com/spreadsheets/d/1WD6rnjVezKFiqLhM85nYCcnkWMOUxORIz8RPvPcL4F4/edit#gid=0